### PR TITLE
hakiri: fix security issue

### PR DESCRIPTION
**Pull request by rishabhs95**

### CODE REVIEW

File: app/controllers/api/base_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Api
   class BaseController < ActionController::Base
-    protect_from_forgery
+    protect_from_forgery(with: :exception)
```
Comment (differentreality) [inline app/controllers/api/base_controller.rb]: Shall we lose the parenthesis?

Comment (rishabhs95) [inline app/controllers/api/base_controller.rb]: Done 👍

**Comment by differentreality** [comment]
:+1: Thanks for the quick fix @rishabhs95 !

**Comment by differentreality** [comment]
Fixes https://github.com/openSUSE/osem/issues/1222

**Pull request merged**